src/components/Search/SearchByNameResults.tsx

```diff
@@ -15,7 +15,7 @@ export const SearchByNameResults = (props: Props) => {
   if (loading) {
     return (
       <div className={styles.searchResultsPanelWrapper}>
-        <Loading show />
+        <Loading show={loading} className={styles.loadingWrapper} />
```

Comment:
Is it necessary to pass `loading` into `<Loading />` when it's in the `loading` branch?

Reply:
updated